index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
If we're assuming ES6 support anyway (parse `import`/`export` and generate `let` and so on), this looks too involved - why not just generate destructuring directly from `require(...)`?